package.json

```diff
@@ -10,32 +10,35 @@
     "migrate": "dotenv -- ley --require tsm"
   },
   "dependencies": {
-    "@apollo/client": "3.8.0-alpha.15",
-    "@apollo/experimental-nextjs-app-support": "^0.1.0",
-    "@apollo/server": "^4.7.1",
-    "@as-integrations/next": "^2.0.0",
+    "@apollo/client": "3.8.0-beta.5",
+    "@apollo/experimental-nextjs-app-support": "^0.3.2",
```

Comment:
`@apollo/client@alpha` is now `@apollo/client@beta` and works only with latest version of `@apollo/experimental-nextjs-app-support`